builtin/game/chat.lua

```diff
@@ -987,6 +1023,7 @@ core.register_chatcommand("clearobjects", {
 		core.clear_objects(options)
 		core.log("action", "Object clearing done.")
 		core.chat_send_all("*** Cleared all objects.")
+		return true
```

Comment:
Probably unrelated too.

Reply:
Since these changes are already in master they should disappear when merging the PR.
It would be better to remove them from it, though.

Reply:
So, should I revert it or keep it as-is?

Reply:
As sfan5 said, these changes are already in master from various other commits. Removing these extraneous changes would make sure the PR only contains your changes.

Reply:
Done.

Reply:
Yes, I understood. I've been there quite a few times as well.

BTW, there are still a lot of extraneous changes that haven't been reverted. All changes up to around L30, for example.